Automattic/wp-calypso
# Add siteSlug prop to CompositeCheckoutContainer

#### Changes proposed in this Pull Request

* All control paths into the `checkout` middleware, where new checkout will be rendered, will have already created a cart on the server. This means that to get the canonical cart data that we need it's enough to call `getCart` with the cart slug.

We also specialize the `setCart` handler to fix the key parameter.

To do:
- [x] Rebase after #37977
- [x] Merge in Filippo's changes from #37941
- [x] Get removeItem to work

#### Testing instructions
Automated tests: `npm run test-packages composite-checkout-wpcom`

Manual test:
- Run calypso with `ENABLE_FEATURES=composite-checkout-wpcom npm start`
- Go through checkout flow for a new site. The selected products should load in the checkout form.
